### PR TITLE
Followed sites docs

### PR DESCRIPTION
**Pull request by learafa**
Documentation for the new followingSites api to follow/unfollow a user's site/sites.
Chnaged the name of the api from followingSites to followedSites

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[6c6188a](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3940/commits/6c6188a389616f18ffaa762868bfce9bde3621e8)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 24 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[6c6188a](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3940/commits/6c6188a389616f18ffaa762868bfce9bde3621e8)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 59 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[6c6188a](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3940/commits/6c6188a389616f18ffaa762868bfce9bde3621e8)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/site-follow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-follow.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fapi%2Fsite-follow.md)
[api-reference/beta/api/site-unfollow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-unfollow.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fapi%2Fsite-unfollow.md)
[api-reference/beta/api/site-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/site.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/resources/site.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/concepts/changelog.md) | :white_check_mark:Succeeded | |

<a name="api-reference%2Fbeta%2Fapi%2Fsite-follow.md"></a>
### [api-reference/beta/api/site-follow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-follow.md)
 - **Line 62**: **[Warning]** Invalid file link:(~/../concepts/errors.md).
 - **Line 67**: **[Warning]** Invalid file link:(~/../concepts/errors.md).

<a name="api-reference%2Fbeta%2Fapi%2Fsite-unfollow.md"></a>
### [api-reference/beta/api/site-unfollow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-unfollow.md)
 - **Line 38**: **[Warning]** Invalid file link:(~/../concepts/errors.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C7%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902071601338006-3940%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=vt6PM0%2F5zlaze5mtnbRit1PFhH54v3JT9DtzqDTtN6I%3D&st=2019-02-07T16%3A00%3A08Z&se=2019-03-10T16%3A05%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/site-follow.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
+---
+author: learafa
+ms.author: learafa
+ms.date: 01/14/2019
+title: Follow Site
```
Comment (Lauragra) [inline api-reference/beta/api/site-follow.md]: Please make sure the ms.date attribute is up to date if you're using it. Also, please add a description attribute; the value can be the summary/introduction. Also note that we only use the author attribute, and the value should be the GitHub name of the content owner. Will this be you going forward? Generally, we should have one author per service/workload.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[5c2a740](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3940/commits/5c2a740a79dedfa6f512a97f67259c44005c6b1e)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 24 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[5c2a740](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3940/commits/5c2a740a79dedfa6f512a97f67259c44005c6b1e)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 59 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[5c2a740](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3940/commits/5c2a740a79dedfa6f512a97f67259c44005c6b1e)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/site-follow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-follow.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fapi%2Fsite-follow.md)
[api-reference/beta/api/site-unfollow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-unfollow.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fapi%2Fsite-unfollow.md)
[api-reference/beta/api/site-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/site.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/resources/site.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/concepts/changelog.md) | :white_check_mark:Succeeded | |

<a name="api-reference%2Fbeta%2Fapi%2Fsite-follow.md"></a>
### [api-reference/beta/api/site-follow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-follow.md)
 - **Line 62**: **[Warning]** Invalid file link:(~/../concepts/errors.md).
 - **Line 67**: **[Warning]** Invalid file link:(~/../concepts/errors.md).

<a name="api-reference%2Fbeta%2Fapi%2Fsite-unfollow.md"></a>
### [api-reference/beta/api/site-unfollow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-unfollow.md)
 - **Line 38**: **[Warning]** Invalid file link:(~/../concepts/errors.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C8%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902081718063056-3940%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=PzUXv8FtOOGW5ORBl3uzWuWcZ1YNa%2FUvbL7b2%2B9OrYk%3D&st=2019-02-08T17%3A16%3A42Z&se=2019-03-11T17%3A21%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[37c7202](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3940/commits/37c7202abfd840b820b36cb2eba249c05b4cea6c)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 24 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[37c7202](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3940/commits/37c7202abfd840b820b36cb2eba249c05b4cea6c)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 59 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[37c7202](https://github.com/microsoftgraph/microsoft-graph-docs/pull/3940/commits/37c7202abfd840b820b36cb2eba249c05b4cea6c)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/site-follow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-follow.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fapi%2Fsite-follow.md)
[api-reference/beta/api/site-unfollow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-unfollow.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fapi%2Fsite-unfollow.md)
[api-reference/beta/api/site-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/site.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/resources/site.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/concepts/changelog.md) | :white_check_mark:Succeeded | |

<a name="api-reference%2Fbeta%2Fapi%2Fsite-follow.md"></a>
### [api-reference/beta/api/site-follow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-follow.md)
 - **Line 62**: **[Warning]** Invalid file link:(~/../concepts/errors.md).
 - **Line 67**: **[Warning]** Invalid file link:(~/../concepts/errors.md).

<a name="api-reference%2Fbeta%2Fapi%2Fsite-unfollow.md"></a>
### [api-reference/beta/api/site-unfollow.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/followedSitesDocs/api-reference/beta/api/site-unfollow.md)
 - **Line 38**: **[Warning]** Invalid file link:(~/../concepts/errors.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C8%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902082020450182-3940%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ezpzgw77jX59Uz6jlsdW63hUMLIlRpT6XPNqFXz1n%2FE%3D&st=2019-02-08T20%3A19%3A16Z&se=2019-03-11T20%3A24%3A16Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/site-follow.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
+---
+author: learafa
+ms.date: 02/08/2019
```
Comment (Lauragra) [inline api-reference/beta/api/site-follow.md]: Please remove the ms.date attribute and values.

File: concepts/changelog.md [hunk 3]
```diff
@@ -306,7 +306,9 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 ### OneDrive and SharePoint APIs
 
 | **Change type** | **Version** | **Description**                          |
-| :-------------- | :---------- | :--------------------------------------- |
+| :-------------- | :---------- | :-------------------------------------- |
```
Comment (Lauragra) [inline concepts/changelog.md]: Were the APIs available in February? For discoverability. I suggest creating a new table in the May section to mention the new APIs.

File: api-reference/beta/api/site-follow.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+In the request body, supply an array of JSON objects with the id parameter mentioned in the table below. 
+
+
+| Name                 | Value  | Description                                                            |
+|:---------------------|:-------|:-----------------------------------------------------------------------|
+|   id                 | string | The [unique identifier](../resources/site.md#id-property) of the item. |
+
+
+## Response body
```
Comment (Lauragra) [inline api-reference/beta/api/site-follow.md]: The standard H2 section is "Response".

File: api-reference/beta/api/site-follow.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+| Delegated (work or school account)     | Sites.ReadWrite.All                         |
+| Delegated (personal Microsoft account) | Not supported.                              |
+| Application                            | Sites.ReadWrite.All                         |
+
+## HTTP request
+
+<!-- { "blockType": "ignored" } -->
+
+```http
+POST https://graph.microsoft.com/v1.0/users/{user-id}/followedSites/add
```
Comment (Lauragra) [inline api-reference/beta/api/site-follow.md]: We don't use the full URL in the HTTP request section, so remove up to /users.

File: api-reference/beta/api/site-unfollow.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
+---
+author: learafa
+ms.date: 02/08/2019
```
Comment (Lauragra) [inline api-reference/beta/api/site-unfollow.md]: Remove date.

File: api-reference/beta/api/site-unfollow.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+| Delegated (work or school account)     | Sites.ReadWrite.All                         |
+| Delegated (personal Microsoft account) | Not supported.                              |
+| Application                            | Sites.ReadWrite.All                         |
+
+## HTTP request
+
+<!-- { "blockType": "ignored" } -->
+
+```http
+POST https://graph.microsoft.com/v1.0/users/{user-id}/followedSites/remove
```
Comment (Lauragra) [inline api-reference/beta/api/site-unfollow.md]: Update URL.

File: api-reference/beta/api/site-unfollow.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+## Request body
+
+In the request body, supply an array of JSON objects with the id parameter mentioned in the table below. 
+
+
+| Name                 | Value  | Description                                                            |
+|:---------------------|:-------|:-----------------------------------------------------------------------|
+|   id                 | string | The [unique identifier](../resources/site.md#id-property) of the item. |
+
+## Response body
```
Comment (Lauragra) [inline api-reference/beta/api/site-unfollow.md]: Change to Response.

**Pull request merged**